media-hover-pointer/index.html

```diff
@@ -0,0 +1,214 @@
+<!doctype html>
+<!--
+Copyright 2014 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <meta name="description" content="Sample illustrating the use of 'any-hover' and 'any-pointer' Media Queries.">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>'any-pointer' and 'any-hover' Media Queries Sample</title>
+
+    <!-- Add to homescreen for Chrome on Android -->
+    <meta name="mobile-web-app-capable" content="yes">
+    <link rel="icon" sizes="192x192" href="../images/touch/chrome-touch-icon-192x192.png">
+
+    <!-- Add to homescreen for Safari on iOS -->
+    <meta name="apple-mobile-web-app-title" content="'any-pointer' and 'any-hover' Media Queries Sample">
+
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+    <link rel="apple-touch-icon-precomposed" href="../images/apple-touch-icon-precomposed.png">
+
+    <!-- Tile icon for Win8 (144x144 + tile color) -->
+    <meta name="msapplication-TileImage" content="images/touch/ms-touch-icon-144x144-precomposed.png">
+    <meta name="msapplication-TileColor" content="#3372DF">
+
+    <link rel="icon" href="../images/favicon.ico">
+
+    <link rel="stylesheet" href="../styles/main.css">
+  </head>
+
+  <body>
+    <h1>'any-pointer' and 'any-hover' Media Queries Sample</h1>
+    <p>Available in <a href="https://www.chromestatus.com/feature/6460705494532096">Chrome 41+</a></p>
+
+    <p>
+      These media queries describe the capabilities of all input devices available to the user as per <a href="http://www.w3.org/TR/2014/WD-mediaqueries-4-20140605/#any-input">the specification</a>.
+      These are similar to the older <code>'pointer'</code> and <code>'hover'</code> media features, except that these only reflect the primary input device.
+    </p>
+    <p>
+      For example, if a user is on a tablet with a connected a mouse, <code>'any-pointer'</code> will be satisfied.
+      If the mouse is then removed, the browser will report <code>'any-pointer'</code> as <code>none</code>.
+    </p>
+    <p>
+      
+    </p>
+
+    <!-- // [START code-block-any-pointer] -->
+    <style>
+      .sized {
+        font-size: 0.5em;
+      }
+      @media (any-pointer: none), (any-pointer: coarse) {
```

Comment:
The text says "This button will be larger where there is no fine pointer available" but that's not what this MQ does (eg. try it on a chromebook pixel, the button is big even though there is a mouse - because 'any-pointer: fine' and 'any-pointer: coarse' are both true).  Maybe you want "not (any-pointer: fine)" instead?